src/AWS/CloudWatchLogs/CloudwatchLogs.purs

```diff
@@ -21,11 +21,15 @@ import Justifill.Fillable (class Fillable)
 import Justifill.Justifiable (class Justifiable)
 import Simple.JSON (class WriteForeign, class ReadForeign, writeImpl)
 import Type.Proxy (Proxy(..))
+import Type.Row (type (+))
 
 foreign import data CloudWatchLogs :: Type
 
 foreign import newCloudWatchLogs :: Foreign -> (Effect CloudWatchLogs)
 
+type MayHasNextToken r
```

Comment:
I'm not a big fan of the name `MayHasNextToken`. I'd rather go with `HasNextToken` 🤷

Reply:
`HasNextToken` wouldn't using an `Maybe` in my opinion :thinking:

Reply:
But `HasNextToken` implies that it can not have a nextToken or not, so i don't think prefixing it with Maybe is needed there